SL_workflow_RNASeq/003.trimming/README.md

```diff
@@ -0,0 +1,16 @@
+Trimming and quality
+====================
+
+Here we trimmed the adaptors with file provided and threshold of 0.0001 quality 28 and minimum length of 40.
+
+The following code has been expressed in a readable style
```

Comment:
Line note: Adding bask slashes in code snippets will allow multi-line bash  to be able to be pasted directly into a terminal

e.g.
```bash
for file in ../001.rawdata/*R1.fq; 
do 
  fname=$(basename $file)
  ## bsub part
  bsub -m genome-s -o log/${fname/_R1.fq/.mcf.log} -e log/${fname/_R1.fq/.mcf.er} \ 
  ## fastqc0mcf part
  fastq-mcf -l 40 -q 28 -t 0.0001 -o ${fname/_R1.fq/.mcf.l40.q28.R1.fastq} 		  \   
   -o ${fname/_R1.fq/.mcf.l40.q28.R2.fastq}                                       \
   adapters.fasta $file ${file/_R1/_R2}
done

```